encryption/mnemonic.ts

```diff
@@ -0,0 +1,37 @@
+import { entropyToMnemonic, mnemonicToEntropy, wordlists } from "bip39";
+import { aesGcmDecrypt, aesGcmEncrypt } from "./aes";
+import { decryptMnemonic, encryptMnemonic } from "@stacks/encryption";
+
+
+/**
+ * @param {string} seed
+ * @returns {string}
+ */
+const seedToEntropy = (seed: string): string => {
```

Comment:
nit: maybe mnemonicToEntropy would be a better name. Or seedPhrase

Reply:
mnemonicToEntropy is declared, so maybe seedPhraseToEntropy